v4/mobile/react-native/FastLink/package.json

```diff
@@ -28,7 +32,9 @@
 		"babel-jest": "^24.9.0",
 		"eslint": "^6.5.1",
 		"jest": "^24.9.0",
-		"metro-react-native-babel-preset": "^0.58.0",
+		"metro-react-native-babel-preset": "^0.59.0",
```

Comment:
Fixes the runtime error:

![Simulator Screenshot - iPhone 15 Pro - 2024-07-15 at 15 48 34](https://github.com/user-attachments/assets/6aa1708b-0990-4c2f-a612-23b641839fe2)